src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelper.cs

```diff
@@ -107,6 +114,15 @@ public ViewDataDictionary ViewData
         }
 
         /// <inheritdoc />
+        public IHtmlEncoder HtmlEncoder { get; }
+
+        /// <inheritdoc />
+        public IUrlEncoder UrlEncoder { get; }
+
+        /// <inheritdoc />
+        public IJavaScriptStringEncoder JavaScriptStringEncoder { get; }
+
+        /// <inheritdoc />
         public IModelMetadataProvider MetadataProvider { get; private set; }
```

Comment:
nit: don't think anything other than the constructor sets this property.  if I'm correct, please remove `private set;` while you're in neighbouring code.